packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,408 +1,414 @@
-## v5.1.0 (TBD)
+## v5.0.1-dev (TBD)
```

Comment:
@shethj you should use "Save without formatting" in VSCode. This addition ends up overwriting the whole git history of the file and makes whitespace changes to the whole thing